### PR TITLE
ign -> gz Upstream Macro Migration : gz-gui

### PR DESCRIPTION
**Pull request by methylDragon**
See: https://github.com/gazebo-tooling/release-tools/issues/737

### CODE REVIEW

**Review by chapulina — APPROVED** [review]
LGTM once upstream PRs are merged and CI is happy 🟢

File: src/Plugin.cc
Comment (chapulina) [inline src/Plugin.cc:187]: Should be up in the `if` together with the warning, and say `ignition-gui`?

File: src/Plugin.cc
Comment (chapulina) [inline src/Plugin.cc:320]: This function is used for saving the config, so I think just changing `ignition` to `gz` is enough. No need to tick-tock. All new saved configs will have `gz`, which is ok. The tick-tock you did for loading old configs that contain `ignition` is enough to tick-tock this feature.

**Review by chapulina — CHANGES_REQUESTED** [review]

File: src/Plugin.cc
Comment (methylDragon) [reply src/Plugin.cc]: https://github.com/gazebosim/gz-gui/pull/413/commits/fe598592f38834a889c2d6bde41bcaee30b3c1d5 un-tick-tocked!

**Comment by chapulina** [comment]
@osrf-jenkins run tests please!

**Pull request merged**
